build.sbt

```
@@ -73,3 +75,10 @@ lazy val `portable-scala-reflect` = crossProject(JSPlatform, JVMPlatform)
     libraryDependencies += "com.novocode" % "junit-interface" % "0.11" % "test",
   )
   .jsConfigure(_.enablePlugins(ScalaJSJUnitPlugin))
+  .nativeSettings(
+    libraryDependencies +=
+      "org.scala-native" %%% "junit-runtime" % "0.4.0" % "test",
+    addCompilerPlugin(
+      "org.scala-native" % "junit-plugin" % "0.4.0" cross CrossVersion.full),
+    mimaPreviousArtifacts := Set.empty // SN just added, no previous artifact
```

We use trailing commas in this build file:
```suggestion
    mimaPreviousArtifacts := Set.empty, // SN just added, no previous artifact
```